utils/python/CIME/case_run.py

```diff
@@ -393,17 +209,21 @@ def case_run(case):
     lid = time.strftime("%y%m%d-%H%M%S")
     os.environ["LID"] = lid
 
-    save_timing_setup_acme(case, lid)
+    save_prerun_provenance(case)
 
     for _ in range(data_assimilation_cycles):
         pre_run_check(case)
         run_model(case)
         post_run_check(case, lid)
         save_logs(case, lid)       # Copy log files back to caseroot
-        get_timings(case, lid)     # Run the getTiming script
+        if case.get_value("CHECK_TIMING") or case.get_value("SAVE_TIMING"):
```

Comment:
Significant change. Things that were being done in this file's get_timings method (directory creation, zipping) have been moved to the core get_timing implementation. Also, notice that we run get_timing if CHECK_TIMING or SAVE_TIMING is true; it used to only run if CHECK_TIMING was true.